### PR TITLE
Scenario request stream

### PR DESCRIPTION
**Pull request by snripa**
Create scenario for testing "RequestStream" scenario at performance environment

### CODE REVIEW

File: ExamplesApi/src/main/java/io/scalecube/gateway/examples/GreetingRequest.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 public class GreetingRequest {
 
   private String text;
+  private Long l;
```
Comment (segabriel) [inline ExamplesApi/src/main/java/io/scalecube/gateway/examples/GreetingRequest.java]: what is 'l'? plz rename it

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSBenchmarkState.java [hunk 2]
```diff
@@ -22,6 +23,8 @@
   private LoopResources loopResources;
   private InetSocketAddress gwAddress;
 
+  StreamRequest streamRequest;
```
Comment (segabriel) [inline Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSBenchmarkState.java]: add private modifier

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    new RSWSBenchmarkState(settings).runWithRampUp(
+        (rampUpTick, state) -> state.createClient(),
+
+        state -> {
+          Timer timer = state.timer("notification.latency");
+          return (executionTick, client) -> {
+            GreetingService service = client.forService(GreetingService.class);
+            return service
+                .requestInfiniteStream(state.streamRequest)
```
Comment (segabriel) [inline Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java]: use a getter to get a request

File: ExamplesApi/src/main/java/io/scalecube/gateway/examples/GreetingRequest.java (hunk 1 above)
Comment (snripa) [reply ExamplesApi/src/main/java/io/scalecube/gateway/examples/GreetingRequest.java]: +

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSBenchmarkState.java (hunk 2 above)
Comment (snripa) [reply Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSBenchmarkState.java]: I use it in package

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java (hunk 3 above)
Comment (snripa) [reply Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java]: k

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSBenchmarkState.java [hunk 4]
```diff
@@ -22,10 +23,16 @@
   private LoopResources loopResources;
   private InetSocketAddress gwAddress;
 
+  private StreamRequest streamRequest;
+
```
Comment (artem-v) [inline Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSBenchmarkState.java]: We dont need this field, just see comment below plz

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        .executionTaskDuration(Duration.ofSeconds(60))
+        .consoleReporterEnabled(true)
+        .durationUnit(TimeUnit.MILLISECONDS)
+        .build();
+
+    new RSWSBenchmarkState(settings).runWithRampUp(
+        (rampUpTick, state) -> state.createClient(),
+
+        state -> {
+          Timer timer = state.timer("notification.latency");
```
Comment (artem-v) [inline Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java]: At this point you can have code like this:
```java
...
    streamRequest = new StreamRequest()
        .setIntervalMillis(settings.executionTaskInterval().toMillis())
        .setMessagesPerInterval(settings.messagesPerExecutionInterval());
...
```
So you can not to have field + getter for it  for  streamRequest object

Comment (snripa) [reply Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/rsocket/RSWSRequestStreamBenchmark.java]: done

**Pull request merged**